hemi/launch.inl

```diff
@@ -36,18 +38,35 @@ void launch(Function f, Arguments... args)
 }
 
 //
+// Automatic parallel launch
+//
+template <typename Function, typename... Arguments>
+void launch(Arguments... args)
```

Comment:
How does one pass a concrete function reference to this launch()? Can you show a working example?